source/Calamari/Kubernetes/Conventions/GatherAndApplyRawYamlConvention.cs

```diff
@@ -41,7 +45,7 @@ public void Install(RunningDeployment deployment)
             var directories = GroupFilesIntoDirectories(deployment, globs, variables);
 
             var resources = new List<Resource>();
-            foreach (var (directory, index) in directories.Select((d,i) => (d,i)))
+            foreach (var (directory, index) in directories.Select((d, i) => (d, i)))
```

Comment:
"Back in my day, we didn't have any of them fancy foreach statements.  I wore an onion on my belt, which was the style at the time..." 😁 
But TBH, it's what you really want here:
```suggestion
            for(int i = 0; i < directories.Count(); i++){
```

Reply:
Otherwise, if you want to keep it set-based, something like this should work fine:
```suggestion
            var resources = directories.SelectMany((directory, index) => ApplyBatchAndReturnResources(index, globs[index], directory))
```